runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -121,6 +122,11 @@ public void onVarbitChange(VarbitChanged event)
 	@Subscribe
 	public void updateConfig(ConfigChanged event)
 	{
+		if (!config.showCharge())
```

Comment:
I would keep this ordered in the same way the config is ordered

Reply:
Decided to reorder the entire function, as it wasn't in order anyway